src/interfaces.py

```diff
@@ -66,6 +66,13 @@ def get_positions(self) -> dict:
     def get_account_balance(self) -> float:
         pass
 
-    
+    @abc.abstractmethod
+    def get_deal_history(self) -> dict:
+        pass
+
+    @abc.abstractmethod
+    def get_account_info(self) -> dict:
```

Comment:
These need to be added to both classes (AccountMT5 & AccountSimulator) if you intend to use them

Reply:
I see that you added these methods to AccountMT5 & AccountSimulator but they are not being called by the flask api currently